exercises/crypto-square/description.md

```diff
@@ -60,6 +62,6 @@ mayoogo
 anouuio
 ntnnlvt
 wttddes
-aohghn
-sseoau
+aohghns
```

Comment:
I am going to trust the canonical data and will instead go fix the specs in the `ecmascript` track. You are also correct that my description change would have misaligned the data. Thank you for responding.